app/components/learnergroup-summary.js

```diff
@@ -15,7 +15,7 @@ export default class LearnergroupSummaryComponent extends Component {
   @tracked currentGroupsSaved = 0;
   @tracked learnerGroupId = null;
   @tracked learnerGroupTitle = null;
-  @Length(2, 100) @tracked location = null;
```

Comment:
I'm worried about his one, doesn't this allow users to specify a location the will not be valid [in the API](https://github.com/ilios/ilios/blob/master/src/Entity/LearnerGroup.php#L79)